includes/amp-helper-functions.php

```diff
@@ -1358,6 +1358,16 @@ function amp_generate_script_hash( $script ) {
 	return 'sha384-' . $hash;
 }
 
+/**
+ * Check if an HTML attribute has a value.
+ *
+ * @param string $attribute Attribute value.
+ * @return bool
+ */
+function amp_is_attribute_empty( $attribute ) {
```

Comment:
Is this necessary? Can't we just use `empty()` instead?

Reply:
As noted in https://github.com/ampproject/amp-wp/pull/3728#discussion_r346164920, `empty()` would return true for the values `0` (int), `"0"` (string) and `0.0` (float), which can result in unforeseen bugs in the future.

Reply:
This is fine for the purposes of `layout`, however. If there is `layout=0` then this seems fine to consider as if it is empty; it will get removed by the tag-and-attribute sanitizer.

I'm just wary off adding new functions to the global namespace unless absolutely needed.

Reply:
Ah I see, I'll refactor this.